NEWS.md

```diff
@@ -86,6 +86,13 @@ This is a major release which rewrites the core code to be asynchronous.
 - **{apt, dnf, pacman}**: `hide_when_uptodate` option is removed and now you can use `format_up_to_date = ""` to hide the block
 - **temperature**: `collapsed` option is removed and now you can use `format_alt = " $icon "` to maintain the behavior
 
+  ```toml
+  [[block]]
+  block = "temperature"
+  format = " $icon{ $max|} "
```

Comment:
I meant just an example in block's documentation (which you've added), I don't think there is a need for this example in NEWS.md